mlos_core/mlos_core/optimizers/flaml_optimizer.py

```diff
@@ -30,9 +30,12 @@ class FlamlOptimizer(BaseOptimizer):
     Wrapper class for FLAML Optimizer: A fast library for AutoML and tuning.
     """
 
+    _METRIC_NAME = "FLAML_score"
```

Comment:
Maybe add comments on why we have this?
e.g.,

```suggestion
    # The name of an internal objective attribute that is calculated as a weighted average of the user provided objective metrics.
    _METRIC_NAME = "FLAML_score"
```